### PR TITLE
added vehicle-license-plate-detection-barrier-0123 model

### PR DESCRIPTION
**Pull request by elenash**
Demo is working with IR.
Accuracy validation passed.
This model is already running in benchmark.

### CODE REVIEW

File: tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+models:
+  - name: vehicle-license-plate-detection-barrier-0123
+
+    launchers:
+      - framework: tf
+        device: CPU
+        model:   object_detection/barrier/tf/0123/model/model.pb.frozen
```
Comment (eaidova) [inline tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml]:
```suggestion
        model:   public/vehicle-license-plate-detection-barrier-0123/model/model.pb.frozen
```

File: tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          - FP16
+        model: public/vehicle-license-plate-detection-barrier-0123/FP16/vehicle-license-plate-detection-barrier-0123.xml
+        weights: public/vehicle-license-plate-detection-barrier-0123/FP16/vehicle-license-plate-detection-barrier-0123.bin
+        adapter: ssd
+
+    datasets:
+      - name: bitvehicle_dataset
+        data_source: BITVehicle/annotation_val/data
+        annotation: bitvehicle.pickle
+        dataset_meta: bitvehicle.json
```
Comment (eaidova) [inline tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml]: could you please change this section on 
```yml
- name: vehicle_license_plate_detection
```
like in config for [vehicle-license-plate-detection-barrier-0106](https://github.com/opencv/open_model_zoo/blob/master/tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0106.yml#L30)

File: models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Average Precision (AP) is defined as an area under the
+[precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve. Validation dataset is BIT-Vehicle.
+
+## Performance
+
+## Inputs
+
+1. name: "input" , shape: [1x3x256x256] - An input image in the format [BxCxHxW],
```
Comment (eaidova) [inline models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md]: for public models we provide 2 section inputs/outputs if model has difference in these layers. As I see this model has `RGB` input in source framework format

File: tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            apply_to: annotation
+            area_ratio: 0.034722222222222224
+
+        metrics:
+          - type: map
+            ignore_difficult: True
+            include_boundaries: False
+            allow_multiple_matches_per_ignored: True
+            distinct_conf: False
+            presenter: print_vector
```
Comment (eaidova) [inline tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml]: please add:
```yml
global_definitions: ../dataset_definitions.yml
```

**Comment by IRDonch** [comment]
Other than the issues already pointed out by others, LGTM.

File: tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml (hunk 2 above)
Comment (elenash) [reply tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml]: @eaidova Should I delete data_source, annotation and dataset_meta fields in this section?

Comment (eaidova) [reply tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml]: yes, all right, and change name to `vehicle_license_plate_detection`

File: models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md (hunk 3 above)
Comment (elenash) [reply models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md]: Please, explain what you mean more specifically

File: tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml (hunk 2 above)
Comment (eaidova) [reply tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml]: it needs for configs reusage in light-weight accuracy validation (on 10 images)

File: models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md (hunk 3 above)
Comment (eaidova) [reply models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md]: when user download public model, he or she makes deal with model in source framework format and after that convert it to IR, preferable way to made it to use converter, but it can be done also manually or even user can use it in source framework format (e.g. load via opencv or in tensorflow). I think we should warn about difference between source framework model and IR if it exists (I mean if model have mean/scale or require image in RGB, when these parameters are a part of IR)

File: tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml (hunk 2 above)
Comment (elenash) [reply tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml]: Thanks!

File: models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    - H - image height
+    - W - image width
+    - C - number of channels
+
+   Expected color order: RGB.
+   Mean values: [127.5,127.5,127.5], scale factor for each channel: 127.5
+
+### Converted Model
+
+An input image, name: "input", shape: [1x3x256x256], format [BxCxHxW],
```
Comment (eaidova) [inline models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md]:
```suggestion
An input image, name: 'input', shape: [1x3x256x256], format [BxCxHxW],
```

Comment (eaidova) [reply models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md]: for alignment with original model input

**Comment by vladimir-dudnik** [comment]
@elenash please address reviewer comment

**Comment by elenash** [comment]
> Please, also extend security_barrier_camera_demo/README.md
Done

File: demos/security_barrier_camera_demo/README.md [hunk 6]
```diff
@@ -2,7 +2,8 @@
 
 This demo showcases Vehicle and License Plate Detection network followed by the Vehicle Attributes Recognition and License Plate Recognition networks applied on top
 of the detection results. You can use a set of the following pre-trained models with the demo:
-* `vehicle-license-plate-detection-barrier-0106`, which is a primary detection network to find the vehicles and license plates
+* `vehicle-license-plate-detection-barrier-0106` or
+* `vehicle-license-plate-detection-barrier-0123`, which are primary detection networks to find the vehicles and license plates
```
Comment (Wovchena) [inline demos/security_barrier_camera_demo/README.md]: I suggest placing them on a single line like `license-plate-recognition-barrier-0001` and`license-plate-recognition-barrier-0007`

File: demos/security_barrier_camera_demo/README.md [hunk 7]
```diff
@@ -90,12 +91,12 @@ To run the demo, you can use public or pre-trained models. To download the pre-t
...
 ```
 
 To do inference for two video inputs using two asynchronous infer request on FPGA with the OpenVINO toolkit pre-trained models, run the following command:
 ```sh
-./security_barrier_camera_demo -i <path_to_video>/inputVideo_0.mp4 <path_to_video>/inputVideo_1.mp4 -m <path_to_model>/vehicle-license-plate-detection-barrier-0106.xml -m_va <path_to_model>/vehicle-attributes-recognition-barrier-0039.xml -m_lpr <path_to_model>/license-plate-recognition-barrier-0001.xml -d HETERO:FPGA,CPU -d_va HETERO:FPGA,CPU -d_lpr HETERO:FPGA,CPU -nireq 2
+./security_barrier_camera_demo -i <path_to_video>/inputVideo_0.mp4 <path_to_video>/inputVideo_1.mp4 -m <path_to_model>/vehicle-license-plate-detection-barrier-0123.xml -m_va <path_to_model>/vehicle-attributes-recognition-barrier-0039.xml -m_lpr <path_to_model>/license-plate-recognition-barrier-0001.xml -d HETERO:FPGA,CPU -d_va HETERO:FPGA,CPU -d_lpr HETERO:FPGA,CPU -nireq 2
```
Comment (Wovchena) [inline demos/security_barrier_camera_demo/README.md]: Is there a strong reason to prefer 0123 over 0106 model? If each model suits different situations (like GFlops vs MParams and mAP tradeoff), I think these 2 example commands shouldn't be changed.

Comment (elenash) [reply demos/security_barrier_camera_demo/README.md]: It seems 123 has better quality, but I'm ot sure. Agree

File: demos/security_barrier_camera_demo/README.md (hunk 6 above)
Comment (elenash) [reply demos/security_barrier_camera_demo/README.md]: Ok

**Comment by elenash** [comment]
Jenknins please retry a build

**Comment by elenash** [comment]
Jenkins please retry a build

**Comment by IRDonch** [comment]
The link checker failed because of `docs.openvinotoolkit.org` being down, so I'm going to ignore it. Merging...
